### ISSUE TITLE
Redis example: Missing sentinel.py

### ISSUE THREAD

**Issue by palonsoro** [kubernetes/kubernetes#17938]
Dockerfile at redis example refers to a sentinel.py that is missing. Where is that file? Is it required?

**Comment by palonsoro** [comment]
If I pull the image from docker hub and inspect it, I see the following contents:

```python
import json
import sys

data = json.load(sys.stdin)

if data and "endpoints" in data and len(data["endpoints"]) > 0:
  host = data["endpoints"][0].split(":")[0]
else:
  host = "localhost"

print "sentinel monitor mymaster %s 6379 2" % host
print "sentinel down-after-milliseconds mymaster 60000"
print "sentinel failover-timeout mymaster 180000"
print "sentinel parallel-syncs mymaster 1"
```

These options seem to be set in this snippet of run.sh:

```sh
sentinel_conf=sentinel.conf

echo "sentinel monitor mymaster ${master} 6379 2" > ${sentinel_conf}
echo "sentinel down-after-milliseconds mymaster 60000" >> ${sentinel_conf}
echo "sentinel failover-timeout mymaster 180000" >> ${sentinel_conf}
echo "sentinel parallel-syncs mymaster 1" >> ${sentinel_conf}
```

So, this file seems to be no longer necessary. Even the Python package itself may also not be necessary, am I right?

**Comment by bgrant0607** [comment]
Fixed by #18141 and #18252
